bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/NetatmoBindingConstants.java

```diff
@@ -50,6 +50,7 @@ public class NetatmoBindingConstants {
     public static final String GROUP_SIGNAL = "signal";
     public static final String GROUP_BATTERY = "battery";
     public static final String GROUP_SECURITY = "security";
+    public static final String GROUP_SECURITY_EVENT = "security-event";
```

Comment:
security-event is just the channel-group. I have now changed that (pushing later), but I found more already existing mistakes regarding channel-groups within the README. Is it required to have channel-groups within the README? Is that useful for the users? As I know the users don't have to know the names of the channel-group to configure items or rules. Maybe we can just remove the information from the README instead of fixing/syncing everything.

Reply:
The group is necessary to identify a channel when linking a channel to an item in a config file. This info is essential.

Reply:
fixed